ipaclient/remote_plugins/__init__.py

```diff
@@ -41,8 +41,13 @@ def _read(self):
         try:
             with open(self._path, 'r') as sc:
                 self._dict = json.load(sc)
-        except EnvironmentError as e:
-            if e.errno != errno.ENOENT:
+        except (EnvironmentError, ValueError) as e:
```

Comment:
Ok, I will update it for the sake of consistency. But there will be no difference in behaviour.